db/data/20240215112358_destroy_icddrb_prescription_drugs.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class DestroyIcddrbPrescriptionDrugs < ActiveRecord::Migration[6.1]
+  ORG_ID = "1bb836eb-9a0b-4646-96ad-7342b53a9155"
+
+  def up
+    unless CountryConfig.current_country?("Bangladesh") && SimpleServer.env.production?
+      print "DestroyIcddrbPrescriptionDrugs is only for production Bangladesh"
```

Comment:
It should print and return, right? Otherwise, it'll continue execution.

Reply:
good catch 🐟 . will fix.